tacacs_plus/__init__.py

```diff
@@ -220,10 +242,10 @@ def __str__(self):
 
 class TACACSAuthenticationStart(object):
 
-    def __init__(self, username, authen_type, data=six.b('')):
+    def __init__(self, username, authen_type, priv_lvl=TAC_PLUS_PRIV_LVL_MIN, data=six.b('')):
```

Comment:
Ah, I see - I misread the diff; you're changing the previously hard-coded value from `TAC_PLUS_PRIV_LVL_MIN`